.github/workflows/codespell.yaml

```diff
@@ -0,0 +1,12 @@
+name: codespell
+
+on: [push, pull_request]
+
+jobs:
+  codespell:
+    name: Codespell
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - run: python3 -m pip install codespell
+      - run: codespell
```

Comment:
Please also add to the `qa`-target in the `Makefile`.

Reply:
Done. `make qa-codespell` just runs codespell, `make qa` runs all checks, including codespell.